Sources/CoreFoundation/CFPlatform.c

```diff
@@ -2300,8 +2304,6 @@ CF_EXPORT int _CFPosixSpawnFileActionsChdir(_CFPosixSpawnFileActionsRef file_act
   // Standardized posix_spawn_file_actions_addchdir version (POSIX.1-2024, June 2024) available in:
   //  - Solaris 11.4 (August 2018)
   //  - NetBSD 10.0 (March 2024)
-  // Currently missing as of:
-  //  - OpenBSD 7.5 (April 2024)
```

Comment:
Just a heads up, this adjustment of the comment made it look confusingly like QNX 8 supports posix_spawn_file_actions_addchdir (it does not). Got confused when I came back to look... fixing in #5192